odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -898,8 +898,7 @@ def test_27_company_dependent(self):
         self.assertEqual(record.with_user(user1).foo, False)
         self.assertEqual(record.with_user(user2).foo, 'default')
 
-        # set field with 'force_company' in context
-        record.with_user(user0).with_context(force_company=company1.id).foo = 'beta'
+        record.with_user(user0).sudo().with_company(company1).foo = 'beta'
```

Comment:
Maybe fix the user definition instead: make them all with `company_ids = [(6, 0, [company0.id, company1.id, company2.id])]`.